<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ld-process-efm</string>
  </property>
  <property name="windowIcon">
   <iconset resource="applicationicons.qrc">
    <normaloff>:/icons/Graphics/ld-process-efm logo 256px.png</normaloff>:/icons/Graphics/ld-process-efm logo 256px.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Control:</string>
    </property>
    <widget class="QPushButton" name="decodePushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Decode</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelPushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_EFM_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_EFM_File">
   <property name="text">
    <string>Open EFM File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="applicationicons.qrc"/>
 </resources>
 <connections/>
</ui>
